### ISSUE TITLE
Add `@@u --url %U @@` to .desktop starter in Flatpak / Snap version

### ISSUE THREAD

**Issue by holterdipolter** [hugolabe/Wike#194]
... so we can open wiki links directly from desktop applications or use it with [Junction](https://apps.gnome.org/de/Junction/).

Thanks for your great application btw!

**Comment by hugolabe** [comment]
Agreed @holterdipolter . I will do some tests as soon as I have time.
